server/tserver/src/main/java/org/apache/accumulo/tserver/log/TabletServerLogger.java

```diff
@@ -271,11 +271,13 @@ public void run() {
         while (!nextLogMaker.isShutdown()) {
           DfsLogger alog = null;
           try {
-            log.debug("Creating next WAL");
+            if (log.isDebugEnabled())
+              log.debug("Creating next WAL");
             alog = new DfsLogger(tserver.getContext(), conf, syncCounter, flushCounter);
             alog.open(tserver.getClientAddressString());
             String fileName = alog.getFileName();
-            log.debug("Created next WAL " + fileName);
+            if (log.isDebugEnabled())
+              log.debug("Created next WAL " + fileName);
```

Comment:
could do `log.debug("Created next WAL {}", fileName);` to avoid string concat when debug not enabled.